static/js/src/advantage/credentials/components/CredExamShop/CredExamShop.tsx

```diff
@@ -18,17 +18,23 @@ const CredExamShop = () => {
     {
       id: "cue-linux-essentials",
       name: "CUE.01 Linux",
+      displayName: "CUE.01 Linux \n ($100 off promotion)",
```

Comment:
I think the strikethrough on the card is enough keep the display name as "CUE.01 Linux".

Reply:
Can you replicate the strikethrough logic on the summary too?

Reply:
And can we also set the display name in local storage so we can use it on the checkout? so we would change the checkout to show display name if available otherwise the product name. Could probably also hide the message in the screenshot bellow:

<img width="652" alt="image" src="https://github.com/canonical/ubuntu.com/assets/8145209/3a4e708c-0e53-480d-9950-a19e9c84d7d5">